test/firmata.test.js

```diff
@@ -1590,7 +1590,7 @@ describe("Board: instances", function() {
 
     it("allows an i2c peripheral's stopTX to be overridden", function(done) {
       // var spy = sandbox.spy(board.transport, "write");
-      var mask = 0b01001000;
+      var mask = 0x48; // 01001000
```

Comment:
This explains the failures I was getting on appveyor today... I didn't dig too deeply though.